generators/openshift/index.js

```diff
@@ -74,6 +74,11 @@ module.exports = class extends BaseGenerator {
                 this.dockerRepositoryName = this.config.get('dockerRepositoryName');
                 this.dockerPushCommand = this.config.get('dockerPushCommand');
                 this.openshiftNamespace = this.config.get('openshiftNamespace');
+                this.mssqlURL = this.config.get('mssqlURL');
+                this.mssqlUser = this.config.get('mssqlUser');
+                this.mssqlPass = this.config.get('mssqlPass');
+                this.mssqlDb = this.config.get('mssqlDb');
```

Comment:
Just follow how it is done for other DB types like MySQL, Postgres etc. No need to do a separate track for MSSQL

Reply:
This and the comment above are to support MSSQL being an external database service. So it doesn't need to generate a new pod and instead needs the connection information.

Reply:
@Spengreb I get it :), but why MSSQL alone? Why not other db types? You can actually change connectionURL being specified in env property of the deployment.yml manually. If you want to make it generic for accessing external services, then it makes sense to do it through Open service broker API. This is in technical preview in OC 3.7.